modules/phase_field/src/postprocessors/GrainTracker.C

```diff
@@ -1415,7 +1415,7 @@ GrainTracker::updateFieldInfo()
         entity_value = _vars[curr_var]->getNodalValue(node);
       }
 
-      if (tmp_map.find(entity) == tmp_map.end() || entity_value > tmp_map[entity])
+      if (entity_value != std::numeric_limits<Real>::lowest() && (tmp_map.find(entity) == tmp_map.end() || entity_value > tmp_map[entity]))
```

Comment:
I'm not assigning lowest. I'm testing to see if _any_ value was assigned at all. This is necessary whenever a phase is passed because it may not enter the branch above leaving the `entity_value` unassigned. To be clear, I'm not reading the gr_op here. That would kind of defeat the purpose of using the grain tracker and just kick the can down the road to the second step. The point is that I want to reconstruct the exact grain structure as is seen in the gr_op variable.